sstsimulator/sst-core
# Implementation of new Python Testing Frameworks

# Preview of the new Python based SST Testing Frameworks.


### Steps
1. clone the git@github.com:allevin/sst-core.git
   * checkout branch `addtesting`
   * build and install sst-core - Note: recommend that installed dir be in the PATH
1. clone the git@github.com:allevin/sst-elements.git
   * checkout branch `addtesting`
   * build and install sst-elements 
1. Run Tests
   * To test SST-Core run `sst-test-core -v`
   * To test SST-Elements run `sst-test-elements -v`
   * Note: Test run results will be output to the `.` directory by default.

[See this link for more details (bottom of page)](https://github.com/allevin/sst-core/tree/addtesting/src/sst/core/testingframework)
